### PR TITLE
Feat/validator

### PR DESCRIPTION
**Pull request by NyaGarcia**

### CODE REVIEW

**Comment by nasdan** [comment]
Apply same feedback than [previous-date validator](https://github.com/Lemoncode/fonk-previous-date-validator/pull/2/files), please

File: src/validator.ts [hunk 1]
```diff
@@ -3,30 +3,39 @@ import {
...
 
-// TODO: Add default message
-let defaultMessage = '';
+export interface CustomArgs {
+  date: Date;
+}
+
+const MISSING_ARGS =
+  'FieldValidationError: date option for date validation is mandatory. Example: { date: new Date() }.';
+const BAD_PARAMETER = 'Value must be a valid Date object.';
```
Comment (nasdan) [inline src/validator.ts]: Remove unused constant

File: examples/js/src/index.js [hunk 2]
```diff
@@ -3,16 +3,20 @@ import { laterDate } from '@lemoncode/fonk-later-date-validator';
...
+      },
+    ],
   },
 };
 
 const formValidation = createFormValidation(validationSchema);
 
-// TODO: Update example values 'test' and/or 10 if needed
 Promise.all([
   formValidation.validateField('myField', 'test'),
```
Comment (nasdan) [inline examples/js/src/index.js]: Replace 'test' by date. To get a normal invalid case.

**Comment by nasdan** [comment]
Great job, thank you. Only two minor changes and i will merge

File: src/validator.ts (hunk 1 above)
Comment (NyaGarcia) [reply src/validator.ts]: Removed the unused BAD_PARAMETER constant => 0f3fd35d6866951994a6717451fce73e8cfe87ad

File: examples/js/src/index.js (hunk 2 above)
Comment (NyaGarcia) [reply examples/js/src/index.js]: Changed test string for Date in index.js => 493a6d0565120d5e5dd436c24e36f58876d92e66
Changed test string for Date in index.ts => 8bb7bfd8c2980ba68f23ba605b5d051b440ff3c8

File: src/validator.spec.ts [hunk 3]
```diff
@@ -1,82 +1,316 @@
-import { validator, setErrorMessage } from './validator';
+import { setErrorMessage, validator } from './validator';
 
-// TODO: Add specs
 describe('fonk-later-date-validator specs', () => {
+  it('should return succeeded validation when value is a valid Date object later than actual Date', () => {
+    const value = new Date(2020, 11, 24, 10, 33, 30, 0);
+    const date = new Date();
```
Comment (nasdan) [inline src/validator.spec.ts]: This spec will fail next year xD. If you want a spec with current date, you could assign date + one day or something like that to `value`

**Pull request merged**